feature/google/src/test/java/BillingWrapperTest.kt

```diff
@@ -1602,72 +1524,6 @@ class BillingWrapperTest {
         assertThat(numCallbacks).isEqualTo(1)
     }
 
-    @Test
```

Comment:
these `queryPurchaseHistoryAsync` tests were harder to keep in common because the mock setup is a little trickier... it's definitely possible, but i wasn't sure it seemed worth it.

my potential solution would require returning a set or custom class from the `mockQueryPurchaseHistory` method -- the BC5 version already returns its builder, but for this test to work we'd also have to return the slot.

anyways, this is kinda the perfect example of how re-using code as a rule might just make the tests less readable. def open to thoughts, or we can plan to return to this discussion once the implementation is closer to being finalized